discojs/discojs-core/src/task/training_information.ts

```diff
@@ -80,7 +80,7 @@ export function isTrainingInformation (raw: unknown): raw is TrainingInformation
   // interdepences on data type
   switch (dataType) {
     case 'image':
-      if (typeof IMAGE_H !== 'number' || typeof IMAGE_W !== 'number') {
+      if ((IMAGE_H !== undefined && typeof IMAGE_H !== 'number') || (IMAGE_W !== undefined && typeof IMAGE_W !== 'number')) {
```

Comment:
I think the best thing to do would be to enforce at least of `RESIZE_IMAGE_*` or `IMAGE_*` to be defined, wdyt?

Reply:
I would say only one between them, what do you think?

Reply:
yes that's I meant, sorry for the typo